include/z64subs.h

```diff
@@ -23,6 +23,20 @@ typedef s32 (*VerifyActor)(struct GlobalContext*, Actor*, Actor*, void*);
 #define SUBS_SHADOW_TEX_HEIGHT 64
 #define SUBS_SHADOW_TEX_SIZE ((s32)sizeof(u8[SUBS_SHADOW_TEX_HEIGHT][SUBS_SHADOW_TEX_WIDTH]))
 
+typedef struct TurnOptions {
+    /* 0x0 */ u16 rotMax; // binary angles
+    /* 0x2 */ u16 slowness; // larger for slower rotation, cannot be 0
```

Comment:
I don't mind slowness, perhaps something more descriptive like slowFactor?

Reply:
Not sure I quite like `slowFactor`, would `slownessFactor` better?